packages/core/src/fhir-to-cda/cda-templates/clinical-document/clinical-document.ts

```diff
@@ -0,0 +1,100 @@
+import { XMLBuilder } from "fast-xml-parser";
+import {
+  buildCodeCE,
+  buildInstanceIdentifier,
+  withNullFlavor,
+  withoutNullFlavorObject,
+} from "../commons";
+import { clinicalDocumentConstants, valueAttribute } from "../constants";
+import {
+  CDAAuthor,
+  CDACustodian,
+  CDARecordTarget,
+  ClinicalDocument,
+} from "../../cda-types/shared-types";
+
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function removeEmptyFields(obj: any): ClinicalDocument {
```

Comment:
this is a very catch-all kinda function.. I think the `any` on the input should stay here, but the output should not be `ClinicalDocument`, right?

Reply:
Idk how specific we can make the output here

Reply:
IDK, but it doesn't make sense to get `any` and return something specific as `ClinicalDocument`. I'd argue against `any` in most scenarios anyways.

Like, it would be better to return `unknown` so the caller is forced to make a decision about the type they expect - since they're the ones who called the function and can take the risk based on the provided param.

Reply:
Yeah, makes sense!